apps/src/lib/ui/accounts/ChangeEmailForm.jsx

```diff
@@ -128,22 +128,34 @@ export default class ChangeEmailForm extends React.Component {
             labelDetails={this.emailOptInLabelDetails()}
             error={validationErrors.emailOptIn}
           >
-            <select
-              value={values.emailOptIn}
-              disabled={disabled}
-              tabIndex="1"
-              onKeyDown={this.onKeyDown}
-              onChange={this.onEmailOptInChange}
-              style={{
-                ...styles.input,
-                width: 100
-              }}
-              ref={el => (this.emailOptInSelect = el)}
-            >
-              <option value="" />
-              <option value="yes">{i18n.yes()}</option>
-              <option value="no">{i18n.no()}</option>
-            </select>
+            <>
```

Comment:
Intentional?

Reply:
Yes. The buttons needed to be grouped under the label in some way. I chose a fragment here, though I don't really understand the pros and cons between fragments and divs in this context so if a div is better I can use that!